reliontomo/protocols/protocol_3d_classify_subtomograms.py

```diff
@@ -214,8 +215,6 @@ def _genCl3dCommand(self):
         cmd += '--iter %i ' % self.nIterations.get()
         if self.useFastSubsets.get():
             cmd += '--fast_subsets '
-        if self.zeroMask.get():
```

Comment:
Why was this option removed? Is it solved the use of 3d or 2d pseudo subtomograms in other place?

Reply:
Because it was duplicated